delius-int/sub-projects/new-tech.tfvars

```diff
@@ -13,33 +13,50 @@ delius_api_lower_cpu_trigger = 5
 delius_api_upper_cpu_trigger = 10
 
 # Override default casenotes Config
-casenotes_conf = {
-
-}
+casenotes_conf = {}
 
 # Override default PDF Generator Config
 pdfgenerator_conf = {}
 
 # Override default Offender API Config
 offenderapi_conf = {
-    env_oracledb_servicename = "POT1NDA_TAF"
+  env_oracledb_servicename = "TSTNDA_TAF"
```

Comment:
Not sure if this is correct Oracle service name sounds like TST stands for Test. I guess a DBA would know

Reply:
That's right, for this environment it should be `INTNDA_TAF`. You can generally find this in the [delius-core config](https://github.com/ministryofjustice/hmpps-env-configs/blob/master/delius-int/sub-projects/delius-core.tfvars#L23).

Reply:
Thanks - updated